README.md

```diff
@@ -16,7 +16,7 @@ Related repositories include:
 
 ## Installing and running Dev Home
 
-> **Note**: Dev Home requires Windows 11 21H2 (build 22000) or later.
+> **Note**: Dev Home requires Windows 11 or Windows 10 version 2004 (code name 20H1 / build number 19041) or newer.
```

Comment:
Thanks for updating this! Although I think Dev Home will technically run on 19041, we officially support 19045. 

Also, now that I'm looking at this, I think we should keep the version info just up here and remove it from below. In fact, I think we can remove the whole "Developer guidance" section, since contributing.md should have that kind of information. (I also don't think you need developer mode enabled to develop Dev Home. You might for sideloading if you build it as an msix, but that's getting in the weeds and I don't think we should include it here.)
```suggestion
> **Note**: Dev Home requires Windows 11 version 21H2 (build 22000) or Windows 10 version 22H2 (build 19045) or newer.
```